contact.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+
+    <!-- Fontawesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="styles/main.css" />
+    <title>Contact Amkam Web Development Bootcamp</title>
+  </head>
+  <body>
+    <!-- Navigation Header -->
+    <header>
+      <!-- Top and Navigation Header -->
+      <div class="bg-secondary top-header">
+        <div class="container d-flex justify-content-end">
+          <div class="d-flex">
+            <ul class="d-flex">
+              <li class="me-3"><i class="fab fa-facebook-f"></i></li>
+              <li><i class="fab fa-twitter"></i></li>
+            </ul>
+            <ul class="d-flex">
+              <li class="me-3 top-header-links">English</li>
+              <li class="me-3 top-header-links">My Page</li>
+              <li class="top-header-links">Logout</li>
+            </ul>
+          </div>
+        </div>
+      </div>
```

Comment:
**Optional Change**
- [ ] This could as well be a HTML5 `nav` element since its intent is to navigate away to social media using links.

Reply:
Okay, let me hit that out.

Reply:
Well solved